equinix/fabric_connection_read_schema.go

```diff
@@ -0,0 +1,708 @@
+			Description: "Priority type",
+		},
+	}
+}
+
+func readPortRedundancySch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"priority": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Priority type",
+		},
+	}
+}
+
+func readChangeLogSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"created_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Key",
+		},
+		"created_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Full Name",
+		},
+		"created_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Email Address",
+		},
+		"created_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by Date and Time",
+		},
+		"updated_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Key",
+		},
+		"updated_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Full Name",
+		},
+		"updated_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Email Address",
+		},
+		"updated_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by Date and Time",
+		},
+		"deleted_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Key",
+		},
+		"deleted_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Full Name",
+		},
+		"deleted_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Email Address",
+		},
+		"deleted_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by Date and Time",
+		},
+	}
+}
+
+func readOrderSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"purchase_order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Purchase order number",
+		},
+		"billing_tier": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Billing tier for connection bandwidth",
+		},
+		"order_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Identification",
+		},
+		"order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Reference Number",
+		},
+	}
+}
+
+func readAccountSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"account_number": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Account Number",
+		},
+		"account_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Account Name",
+		},
+		"org_id": {
+			Type:        schema.TypeInt,
```

Comment:
Just checking - `TypeInt` not `TypeString`? (I see `global_org_id` and `global_cust_id` are TypeString, so I'm wondering about the API consistency here in terms of names hinting the type).

Reply:
Yes it is int, this data is from legacy system. Connection API is a consumer to the data.